Doc/c-api/apiabiversion.rst

```diff
@@ -58,5 +58,15 @@ See :ref:`stable` for a discussion of API and ABI stability across versions.
    Thus ``3.4.1a2`` is hexversion ``0x030401a2`` and ``3.10.0`` is
    hexversion ``0x030a00f0``.
 
+   Starting with Python 3.11, this information is also available via the
+   exported symbol `Py_Version`.
```

Comment:
Can you please add a similar note in Py_GetVersion() documentation? See Doc/c-api/init.rst.